golem-component-service-base/tests/db/migration/postgres/001__init.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE components
```

Comment:
Why do we have dedicated migrations for tests? I believe there should be only 1 set of migrations

Reply:
tests are in base module, maybe we can have migration definition there, and then somehow copy them to service module

Reply:
Alternatively you can explicitly specify path to migrations in the service module. It is ugly, but should work.